### ISSUE TITLE
Throws exception with tag pushes

### ISSUE THREAD

**Issue by faizanakram99** [trilom/file-changes-action#95]
**Describe the bug**
Throws exception with tag pushes

**Workflow**
Push a tag, it will throw exception instead of showing files added, modified, deleted as empty

**Expected behavior**
Not to throw any exception

**Additional context**
Here is the part which might be interesting
```
      "headers": {
        "accept": "application/vnd.github.v3+json",
        "user-agent": "octokit.js/16.43.1 Node.js/12.13.1 (Linux 5.0; x64)",
        "authorization": "token [REDACTED]"
      },
      "request": {}
    },
    "documentation_url": "https://developer.github.com/v3/repos/commits/#compare-two-commits"
  }
}
(node:3600) UnhandledPromiseRejectionWarning
```

I am using output format as `' '`

**Comment by trilom** [comment]
thank you, I don't think I have done any work with this and tags.

**Comment by trilom** [comment]
Adding my notes/thoughts.

Based on what you provided, the issue is happening with the [github api call here.](https://github.com/trilom/file-changes-action/blob/2d1de7f0f5fd986e8682089323970a3870cdbfe3/src/GithubHelper.ts#L87-L92)

You can find all the tests for [that function here.](https://github.com/trilom/file-changes-action/blob/2d1de7f0f5fd986e8682089323970a3870cdbfe3/src/tests/GithubHelper.test.ts#L131-L210).

Find a way to write a test for this problem and then write the code around that API call to handle it, or elsewhere.

**Comment by trilom** [comment]
However this might be resolved in #99 - please let me know if otherwise.
